client/python/kserve-api/samples/kfs_grpc_model_ready.py

```diff
@@ -0,0 +1,44 @@
+#
+# Copyright (c) 2022 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+import grpc
+import argparse
+
+from tritonclient.grpc import service_pb2
+from tritonclient.grpc import service_pb2_grpc
+
+if __name__ == '__main__':
+    parser = argparse.ArgumentParser(description='Sends requests via TFS gRPC API using images in numpy format. '
+                                                 'It displays performance statistics and optionally the model accuracy')
+    parser.add_argument('--grpc_address',required=False, default='localhost',  help='Specify url to grpc service. default:localhost')
+    parser.add_argument('--grpc_port',required=False, default=9000, help='Specify port to grpc service. default: 9000')
+    parser.add_argument('--model_name', default='resnet', help='Define model name, must be same as is in service. default: resnet',
+                        dest='model_name')
+    parser.add_argument('--model_version', default="",
+                        help='Define model version. If not specified the default version will be taken from model server',
+                        dest='model_version')
+
+    args = vars(parser.parse_args())
+
+    address = "{}:{}".format(args['grpc_address'],args['grpc_port'])

```

Comment:
Since address is merged from `grpc_address` and `grpc_port `maybe we could have a single parameter taking a whole expression `<addres>:<port>` like:
`--service_url localhost:9000` ?